pkgs/development/r-modules/default.nix

```diff
@@ -472,7 +472,10 @@ let
     VariantAnnotation = with pkgs; [ zlib.dev curl.dev ];
     snpStats = [ pkgs.zlib.dev ];
     vcfppR = [ pkgs.curl.dev pkgs.bzip2 pkgs.zlib.dev pkgs.xz];
-    hdf5r = [ pkgs.hdf5.dev ];
+    hdf5r = [ (pkgs.hdf5_1_10.dev.overrideAttrs (prev: new: {
```

Comment:
I think if we keep this, this should likely go all the way down starting at line 944 (where packages with overrides are defined)